modules/bootstrapped/src/generated/smithy4s/example/StructureWithRefinedMember.scala

```diff
@@ -15,7 +15,7 @@ object StructureWithRefinedMember extends ShapeTag.Companion[StructureWithRefine
   val hints: Hints = Hints.empty
 
   implicit val schema: Schema[StructureWithRefinedMember] = struct(
-    int.refined[smithy4s.refined.Age](smithy4s.example.AgeFormat()).optional[StructureWithRefinedMember]("otherAge", _.otherAge).addHints(smithy4s.example.AgeFormat()),
```

Comment:
is it expected that his addHints is gone?

Reply:
Yeah that reason for this is that the hint is rendered inside of the refined schema already. We already did the same thing for NewTypes (see how they are rendered) so this is just bringing it into parity